File: packages/jsii-rosetta/test/translations/classes/class_with_props_argument.cs
class MyClassProps
{
    public string Prop1 { get; set; }
    public int Prop2 { get; set; }
}

class MyClass : cdk.SomeOtherClass
{
    public MyClass(cdk.Construct scope, string id, MyClassProps props) : base(scope, id, props)
    {

        Console.WriteLine(props.Prop1);
    }
}

File: packages/jsii-rosetta/test/translations/classes/class_with_props_argument.go
type myClassProps struct {
	prop1 *string
	prop2 *f64
}

type myClass struct {
	cdk.SomeOtherClass
}

func newMyClass(scope cdk.Construct, id *string, props myClassProps) *myClass {
	this := &myClass{}
	cdk.NewSomeOtherClass_Override(this, scope, id, props)

	fmt.Println(*props.prop1)
	return this
}
